core/geometry/src/test/topology/ChainCollector.test.ts

```diff
@@ -240,3 +242,39 @@ describe("ChainCollector", () => {
     expect(ck.getNumErrors()).equals(0);
   });
 });
+
```

Comment:
"ChainMergeVariants" test is moved here form PolyfaceQuery.test.ts with no change.